lib/pdk/cli/exec.rb

```diff
@@ -55,6 +55,7 @@ class Command
         attr_reader :argv
         attr_reader :context
         attr_accessor :timeout
+        attr_accessor :environment
```

Comment:
👍 on the change itself, it does feel like it could be a separate commit though, as it adds a feature unrelated to the "puppetlabs_spec_helper invocation for unit tests" mentioned in the commit message.